angular-workspace/projects/ni/nimble-angular/table-column/anchor/nimble-table-column-anchor-router-link-with-href.directive.ts

```diff
@@ -11,8 +12,10 @@ import type { TableColumnAnchorCellView } from '@ni/nimble-components/dist/esm/t
     selector: 'nimble-table-column-anchor[nimbleRouterLink]'
 })
 export class NimbleTableColumnAnchorRouterLinkWithHrefDirective extends RouterLinkWithHref {
-    public set routerLink(_commands: unknown[] | string | null | undefined) {
-        throw new Error('Directly configuring the routerLink url is not supported. The router url is configured via the href-field-name of the column.');
+    public set nimbleRouterLink(commands: unknown[] | string | null | undefined) {
+        if (commands !== undefined && commands !== null) {
```

Comment:
Can you explain the need for narrowing the scope for when we throw the error? We're now saying it's fine to set the `nimbleRouterLink` to `null` or `undefined`, right? Why? If so, it seems like something that would merit a comment in code.

Reply:
Formerly, we were throwing an error for anyone try to set `routerLink` -- not `nimbleRouterLink` -- but that was actually unnecessary, because we have a separate directive that errors for that case. I turned this into an error for the user trying to set `nimbleRouterLink` to a value. We want to support adding the attribute without a value, i.e. `<nimble-table-column-anchor nimbleRouterLink>`. You can't explicitly set the attribute value to `null` or `undefined` via an assignment in the html, can you? Really, we just want to let the user know that the typical way of setting `routerLink`/`nimbleRouterLink` by assigning a URL is not the way it works with the anchor column.

Does that make sense? Do you still think I need a comment explaining that?

Reply:
In Angular, it would be perfectly valid to set `nimbleRouterLink` to `null` or `undefined` in HTML:

```html
<nimble-table-column-anchor [nimbleRouterLink]="undefined"></nimble-table-column-anchor>
```

Now, that wouldn't necessarily be valid in an Angular application since `nimbleRouterLink` isn't called out as an `@Input`.

Reply:
Ah right. If binding, I guess "null" and "undefined" are not interpreted as strings. Still, I think we have to allow `null` and `undefined` (or maybe just `undefined`?) to support the expected use case of providing `nimbleRouterLink` without anything assigned.